libafl/src/executors/forkserver.rs

```diff
@@ -499,8 +499,14 @@ impl Forkserver {
     /// Read bytes of any length from the st pipe
     pub fn read_st_size(&mut self, size: usize) -> Result<(usize, Vec<u8>), Error> {
         let mut buf = vec![0; size];
-
-        let rlen = self.st_pipe.read(&mut buf)?;
+        let mut rlen = 0;
+        while rlen < size {
+            let bytes_read = self.st_pipe.read(&mut buf[rlen..])?;
```

Comment:
This can use [read_exact](https://doc.rust-lang.org/std/io/trait.Read.html#method.read_exact) instead

Reply:
Yeah. True. I replaced it with `read_exact`.

BTW I found returning the size of the bytes is redundant because it will always return  the expected size when it succeeds. Should we update the return type as well?

Reply:
Sounds good. Also removes some checks.